src/main/java/com/farao_community/farao/gridcapa/data_bridge/FileMetadataConfiguration.java

```diff
@@ -22,6 +22,8 @@ public class FileMetadataConfiguration {
     private String timeValidity;
     @Value("${data-bridge.file-regex}")
     private String fileRegex;
+    @Value("${data-bridge.do-unzip}")
```

Comment:
I suggest that we set a default value : 
 @Value("${data-bridge.do-unzip:true}")